src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { createRef } from 'react'
+import { displayTo } from '@/helpers/visibilityClassnames'
+import { useState } from 'react'
+import Arrow from '@/icons/Arrow'
+import Menu from '@/components/Dropdown/Menu'
+import Option from '@/components/Dropdown/Option'
+import classnames, {
+  alignItems,
+  backgroundClip,
+  backgroundImage,
+  display,
+  fontSize,
+  fontWeight,
+  gap,
+  gradientColorStops,
+  justifyContent,
+  margin,
+  opacity,
+  position,
+  textColor,
+  width,
+} from 'classnames/tailwind'
+import useClickOutside from '@/hooks/useClickOutside'
+
+const gradientText = classnames(
+  textColor('text-transparent'),
+  backgroundImage('bg-gradient-to-r'),
+  backgroundClip('bg-clip-text'),
+  gradientColorStops('from-secondary', 'to-accent'),
+  fontWeight('font-bold')
+)
+const textStyles = (colorfulCurrentValue?: boolean) =>
+  classnames(
+    fontSize('text-sm'),
+    colorfulCurrentValue ? gradientText : undefined
+  )
+const button = classnames(
+  display('flex'),
+  justifyContent('justify-start'),
+  alignItems('items-center'),
+  gap('gap-x-2'),
+  opacity('disabled:opacity-30')
+)
+const container = (displayBeforeMd?: boolean) =>
+  classnames(
+    position('relative'),
+    margin('my-2'),
+    displayBeforeMd ? displayTo('md') : undefined
+  )
+
+export default function ({
+  disabled,
+  currentValue,
+  options,
+  onChange,
+  displayBeforeMd,
+  staticPlaceholder,
+  fitToItemSize,
+  colorfulCurrentValue,
+}: {
+  currentValue: string
+  options: Option[]
+  onChange: (selectedValue: string) => void
+  disabled?: boolean
+  staticPlaceholder?: string
+  fitToItemSize?: boolean
+  colorfulCurrentValue?: boolean
+  displayBeforeMd?: boolean
```

Comment:
It doesn't need here because if you want to display component in specific resolutions, you should wrap it inside it